src/sage/modular/modsym/space.py

```diff
@@ -46,7 +47,7 @@
 from sage.modular.hecke.module import HeckeModule_free_module
 from sage.modular.modsym.element import ModularSymbolsElement
 
-from . import hecke_operator
+lazy_import('sage.modular.modsym', 'hecke_operator')
 
```

Comment:
Why is this lazy-imported?

Reply:
IIRC, via .hecke it depends on .arithgroup, which I had some difficulties with

Reply:
OK.